### DISCUSSION TITLE
Enable Virtualization Technology in BIOS

### DISCUSSION THREAD

**Discussion by tuannda94** [utmapp/UTM#5673] [Windows]
**Describe the issue**  
Thank for teams. I'm using UTM for run apk file with LDPlayer Emulator. I opened BIOS before start windows but i can't find Virtualization Technology in BIOS config. Can you help me?

**Configuration**  
* UTM Version: 4.3.5 (87)
* macOS Version: Ventura 13.2.1
* Mac Chip: M1

<img width="1680" alt="Screenshot 2023-09-07 at 09 55 58" src="https://github.com/utmapp/UTM/assets/26887039/35a33d8d-9ed9-4698-a86e-114671edfdd8">

Thank for support! This is awesome project!

**Comment by osy** [comment]
This is not supported

**Reply by am11** [reply]
According to https://apple.stackexchange.com/a/467030, nested virtualization is supported on Apple Silicon. Might be good to explore if this option can be applied to UTM. These days, many stacks are requiring WSL/Hyper-V/docker on Windows.

**Reply by am11** [reply]
Ah nvm, actually it's not officially supported yet. That Asahi Linux bit made me wonder if there is a workaround but may not be available from within the macOS environment until Apple officially supports it. 🥲

**Comment by parthdevloper** [comment]
+1 for docker
